db/migrate/20240908091138_add_service_name_to_active_storage_blobs.active_storage.rb

```diff
@@ -0,0 +1,22 @@
+# This migration comes from active_storage (originally 20190112182829)
```

Comment:
Hmm, not a problem raised by this PR, but do we use activestorage for anything? Maybe we should turn it off, and this wouldn't be necessary.